voidcrew/modules/overmap/code/controllers/subsystem/overmap.dm

```diff
@@ -26,11 +28,19 @@ SUBSYSTEM_DEF(overmap)
 	///List of all simulated ships
 	var/list/simulated_ships = list()
 
+	#ifdef UNIT_TESTS
```

Comment:
Don't we want a ship to spawn during unit testing?

Reply:
I had this define backwards, its supposed to spawn a specific ship only during unit tests